lib/tower_cli/models/base.py

```diff
@@ -828,7 +837,7 @@ def status(self, pk, detail=False):
                   help='If provided, this command (not the job) will time out '
                        'after the given number of seconds.')
     def monitor(self, pk, min_interval=1, max_interval=30,
-                timeout=None, outfile=sys.stdout):
+                timeout=None, outfile=sys.stdout, **kwargs):
```

Comment:
Note: Adding kwargs here enables use for resources that have fields declared on them. Even if those fields are not specified, they are passed with other utilities with None values. This is particularly relevant for the ad hoc commands, because there is a greater need for fields (more similar to job template), rather than just command line flags.